index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio-Mobile</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <div class="logo"><a href="#">StevenCjay</a></div>
+        <div class="logo2"><img src="img/Icons/Union.png" alt="" /></div>
+      </nav>
+    </header>
+    <main>
+      <section class="profile">
+        <h1>Hello I am Steven</h1>
+        <h2>I am a Software Developer</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, do not hestiate to contact me.
+        </p>
+        <ul>
+          <li>
+            <a href="https://github.com/emmanuelsteven"
+              ><ion-icon name="logo-github"></ion-icon
+            ></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/CjayStev"
+              ><ion-icon name="logo-twitter"></ion-icon
+            ></a>
+          </li>
+          <li>
+            <a href="https://web.facebook.com/chijokes/"
+              ><ion-icon name="logo-facebook"></ion-icon
+            ></a>
+          </li>
+          <li>
+            <a href="https://www.instagram.com/cjaystev22/"
+              ><ion-icon name="logo-instagram"></ion-icon
+            ></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/steven-emmanuel-75a03a141/"
+              ><ion-icon name="logo-linkedin"></ion-icon
+            ></a>
```

Comment:
As mentioned by @aamirkhan2478, kindly make sure to use the icons from the Figma design.
One important aspect that we need to learn here at Microverse is to understand how to parse a Figma design to create a UI.
If it is required to use the elements from there, let's make sure we do that.

After we finish the module, and all the milestones are approved, we can change the app according to our taste.